src/FFTW.jl

```diff
@@ -25,6 +25,7 @@ end
 
 # MKL provides its own FFTW
 const fftw_vendor = occursin("mkl_rt", libfftw3) ? :mkl : :fftw
+const fftw_last_num_threads = Ref(Cint(1))
```

Comment:
I would just call it `_last_num_threads` — start it with an underscore to emphasize that this is an internal thing that should never be used outside this module, and omit the `fftw` prefix since it isn't needed for FFTW itself and we are already in the module namespace.   And only define it `if fftw_vendor == :mkl`

Reply:
Done